src/codeu/chat/client/commandline/Chat.java

```diff
@@ -40,6 +43,44 @@ public Chat(Context context) {
 		this.panels.push(createRootPanel(context));
 	}
 
+	// RECREATE 
+	//
+	// Recreates the server's last known state to display for the client 
```

Comment:
it's saved from last time and this function just reminds you of that fact since it prints out all the information that has been saved from the last set of conversations that was hosted by the server